NEWS.md

```diff
@@ -4,6 +4,15 @@ If you read this from a place other than <https://mc-stan.org/projpred/news/inde
 
 # projpred 2.5.0.9000
 
+## Major changes
+
+* Until now, `solution_terms.vsel()` was the only function for extracting a solution path from `vsel` objects (returned by `varsel()` or `cv_varsel()`), namely the solution path from the full-data search. This function is unchanged by this release, to avoid breaking changes for users. However, in anticipation of a larger overhaul of the **projpred** user interface, this release introduces some new functions that are useful for accessing and investigating solution paths (which are now termed *predictor rankings* by these new functions, a term that is hopefully easier to grasp for new users):
```

Comment:
I think this could be ok, but you could ask additional comments for the term predictor ranking from the workflow group and in Stan slack

Reply:
Yes, I'll do that. Thanks.

Reply:
I haven't received objections against "predictor ranking", so I think we can go for that term.